README.md

```diff
@@ -16,7 +16,7 @@ Throughout my career there has always been a disconnect between the documentatio
 
 We will need a few tools for this guide. We will walk through how to install `helm` and `kubectl`.
 
-Or [Watch the video](https://youtu.be/oM-6sd4KSmA).
+Or [Watch the video](https://youtu.be/ONes6pv_9J4).  The original but outdated [video is here.](https://youtu.be/oM-6sd4KSmA).
```

Comment:
Can you move "The original..." to a new line?

Reply:
Done.